core/che-core-api-core/src/test/java/org/eclipse/che/api/core/jsonrpc/commons/JsonRpcMessageReceiverTest.java

```diff
@@ -30,7 +30,7 @@
 public class JsonRpcMessageReceiverTest {
 
   static final String MESSAGE = "message";
-  static final String ENDPOINT_ID = "endpoint-id";
+  static final String ENDPOINT_ID = "client-id<-:->endpoint-id";
```

Comment:
Shouldn't this string use `org.eclipse.che.api.core.websocket.impl.WebsocketIdService.SEPARATOR` instead of a fixed value?